deploy/olm-operators.configmap.yaml

```diff
@@ -292,11 +292,11 @@ data:
         minKubeVersion: "1.10.0"
         maturity: alpha
```

Comment:
Not to worry about changing this. I'm changing this file in another PR I'm working on.